### ISSUE TITLE
Adjust the formatting of code

### ISSUE THREAD

**Issue by sojusnik** [peter88213/zim2obsidian#20]
This script currently converts inline code with two ``  and code blocks with tabs, whereas it should do it as mentioned [here](https://help.obsidian.md/Editing+and+formatting/Basic+formatting+syntax#Code) or according to the following examples:

```
Zim: An example ''with code'' inside a sentence.
Obsidian: An example `with code` inside a sentence.
```

A Zim code block:
```
'''
This is a code block, a code block this is, a code block this is, a code block this is, a code block this is, a code block this is, a code block this is, a code block this is, a code block this is, a code block this is, a code block this is, a code block this is.
'''
```

The upper and lower `'''` from above should be converted to ``` in Obsidian.

While the [manual](https://help.obsidian.md/Editing+and+formatting/Basic+formatting+syntax#Code+blocks) says that a code block can also start with a tab (or 4 spaces), it's rendered weirdly, so therefore the formatting with ``` should be preferred, as shown here:

![grafik](https://github.com/peter88213/zim2obsidian/assets/1245718/5e2e1c3a-025e-4b04-be14-efb8e3575ea1)

While with a tab at the beginning, it looks like
![grafik](https://github.com/peter88213/zim2obsidian/assets/1245718/21db7272-1c75-4136-9310-473ef5be3d82)

**Comment by peter88213** [comment]
To be precise, the *zim2obsidian* script does not change at all the inline code formatting as exported by Zim. 
I guess, the problem is that Zim doesn't export "fenced" code blocks, see [issue 1718](https://github.com/zim-desktop-wiki/zim-desktop-wiki/issues/1718)
At the moment, it is marked as an "open issue" in the [code](https://github.com/zim-desktop-wiki/zim-desktop-wiki/blob/develop/zim/formats/markdown.py#L97).

If you want to do it with *zim2obsidian*, you have to enhance the code for placing the "fences" around lines beginning with a tab character. This works only, if the tab characters that signify code block lines are not already escaped by *subst_indent.py*. 

I think, this problem is best solved by converting the original Zim notes as discussed in #8.

**Comment by peter88213** [comment]
With version 0.11.6, the *zim2obsidian.py* script supports code blocks that are "fenced" with backticks (see #21). This is not specified with Zim, but it may used as a workaround. Same for "inline code" embraced with single backticks 8see #22).

**Comment by peter88213** [comment]
Closed with version 0.13.0, see: https://github.com/peter88213/zim2obsidian/discussions/15#discussioncomment-8918078
